main/store.js

```diff
@@ -1,21 +1,27 @@
 'use strict'
 
 const Store = require('electron-store')
-const electron = require('electron')
+const { randomUUID } = require('crypto')
 
-const defaults = {
-  // https://github.com/sindresorhus/electron-store#defaults
-  language: (electron.app).getLocale() // useful for i18n
-}
+const store = new Store({
+  defaults: {
+    stationID: randomUUID()
+  }
+})
 
-/** @type {import('electron-store').Options<{language: string}>["migrations"]} */
-const migrations = {
-  // https://github.com/sindresorhus/electron-store#migrations
+const keys = {
+  stationID: 'stationID'
```

Comment:
in contrast to other store configs, I decided not to start variables with an uppercase letter, since those are usually reserved for classes and constants. Wdyt, @bajtos?

Reply:
wouldn't constants be all upper case `STATION_ID` more typically? I'm fine with either as long as it follows a common convention

Reply:
Yeah, all upper case is a common pattern for naming constants.

Recently, I read somewhere an opinion advocating for using PascalCase for constants and the arguments made sense, so I wanted to try it out in practice. I quite like how the code reads this way, but it's subjective.

I don't mind either way: `stationID`, `STATION_ID`, `StationID` or `StationId`. (I slightly prefer `Id` over `ID` but not a big deal.)

I think it would be good to be consistent. Agree on one style, commit to the decision and move on. If possible, configure a linter rule to enforce the style.

Reply:
I'm fine with either I think, PascalCase is easier to read, but I only want to go there if this either is a common pattern in the ecosystem, or really is a significant improvement. Otherwise I think the "differentness" isn't worth it. This is a very common argument for me, what do you think about this?

Also +1 to enforcing this through lint rules :)